drivers/cc110x/cc110x-netdev2.c

```diff
@@ -0,0 +1,218 @@
+/*
+ * Copyright (C) 2014 Freie Universität Berlin
+ *               2015 Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_cc110x
+ * @{
+ * @file
+ * @brief       Implementation of netdev2 interface for cc110x
+ *
+ * @author      Fabian Nack <nack@inf.fu-berlin.de>
+ * @author      Kaspar Schleiser <kaspar@schleiser.de>
+ * @}
+ */
+
+#include <assert.h>
+#include <stdlib.h>
+#include <string.h>
+#include <errno.h>
+
+#include "cc110x.h"
+#include "cc110x/cc110x-netdev2.h"
+#include "cc110x/cc110x-internal.h"
+#include "cc110x/cc110x-interface.h"
+#include "net/eui64.h"
+
+#include "periph/cpuid.h"
+#include "periph/gpio.h"
+#include "net/netdev2.h"
+#include "net/gnrc/nettype.h"
+
+#define ENABLE_DEBUG    (0)
+#include "debug.h"
+
+static int _send(netdev2_t *dev, const struct iovec *vector, int count)
+{
+    DEBUG("%s:%u\n", __func__, __LINE__);
+
+    netdev2_cc110x_t *netdev2_cc110x = (netdev2_cc110x_t*) dev;
+    cc110x_pkt_t *cc110x_pkt = vector[0].iov_base;
```

Comment:
Yes, the driver currently accepts only that. But gnrc_cc110x converts a full pktsnip into one iovec.